release-notes/stable.yml

```diff
@@ -1,4 +1,10 @@
 releases:
+  42.20250705.3.0:
+    issues:
+      - text: "Implement Proxmox VE (`proxmoxve`) Support"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1652"
+      - text: "Implement Oracle Cloud Infrastructure support"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1967"
```

Comment:
I guess let's drop this announcement from here since the [stable image is broken](https://github.com/coreos/fedora-coreos-tracker/issues/1967#issuecomment-3111562038).